src/xenia/kernel/xam/xam_content.cc

```diff
@@ -13,7 +13,7 @@
 #include "xenia/kernel/xam/xam_private.h"
 #include "xenia/kernel/xenumerator.h"
 #include "xenia/xbox.h"
-
+DEFINE_int32(license_mask, 0, "Toggle license for XBLA games", "Kernel");
```

Comment:
Description needs to be adjusted. It's not a toggle, it's a bit set of activated licenses. Something like `Set license mask for activated content`. I'm not sure how to make this easily understood though.

Category should also be `Content`, not `Kernel`.

Reply:
Changed to `Set license mask for activated content` and added value description from @Triang3l